### PR TITLE
[Enhancement] - Deco legacy transformers and train InceptionV1 to meet training target

### PR DESCRIPTION
**Pull request by wzhongyuan**
## What changes were proposed in this pull request?
Deco legacy transformers and train InceptionV1 to meet training target

## How was this patch tested?
UT

## Related links or issues (optional)
fixed https://github.com/intel-analytics/BigDL/issues/XXX

### CODE REVIEW

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/ImageNet2012.scala [hunk 1]
```diff
@@ -36,14 +36,18 @@ object ImageNet2012 {
     classNumber: Int
   )
   : DataSet[MiniBatch[Float]] = {
-    DataSet.SeqFileFolder.files(path, sc, classNumber).transform(
-      MTLabeledBGRImgToBatch[ByteRecord](
+    DataSet.SeqFileFolder.filesToImageFeatureDataset(path, sc, 1000).transform(
```
Comment (zhangxiaoli73) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/ImageNet2012.scala]: classNumber may be better than 1000

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/ImageNet2012.scala [hunk 2]
```diff
@@ -72,14 +76,19 @@ object ImageNet2012Val {
      classNumber: Int
    )
    : DataSet[MiniBatch[Float]] = {
-     DataSet.SeqFileFolder.files(path, sc, classNumber).transform(
-       MTLabeledBGRImgToBatch[ByteRecord](
+
+     DataSet.SeqFileFolder.filesToImageFeatureDataset(path, sc, 1000).transform(
```
Comment (zhangxiaoli73) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/ImageNet2012.scala]: above

Comment (wzhongyuan) [reply spark/dl/src/main/scala/com/intel/analytics/bigdl/models/inception/ImageNet2012.scala]: Refined, but this is specifically for ImageNet2012, not very common method

**Comment by zhangxiaoli73** [comment]
LGTM

**Comment by wzhongyuan** [comment]
Jenkins passed  http://mlt-bddl01.sh.intel.com:8080/job/BigDL-PRVN-lite/246/

**Pull request merged**